pages/performances/index.js

```diff
@@ -18,62 +22,302 @@ import styles from '@styles/pages/Performances.module.scss'; // Page styles
 // Temp constants
 import data, { columns } from '../../data/mockParticipants';
 
+const PAGE_SIZE = 3; // Rows per page
```

Comment:
I'm assuming this changed based on which page we're on? Maybe this was copied over wrong from the setting page?

Reply:
This is a constant that sets the maximum number of rows per page. I set the value to 3 just for testing, but we can set it to whatever is needed.